### PR TITLE
remove TODO

### PR DESCRIPTION
**Pull request by amityadav0**

### CODE REVIEW

File: packages/algob/test/internal/core/runtime-environment.ts [hunk 1]
```diff
@@ -313,23 +313,6 @@ describe("Environment", () => {
         );
       }, ERRORS.NETWORK.CONFIG_NOT_FOUND);
     });
-
-    // TODO:RZ -- network may need to be nullable
-    // or default parameter should be provided as default in CLI
```
Comment (amityadav0) [inline packages/algob/test/internal/core/runtime-environment.ts:318]: does the network needs to be nullable ?

if not, these lines are removed

**Comment by amityadav0** [comment]
Other subtasks were already done

File: packages/algob/test/internal/core/runtime-environment.ts (hunk 1 above)
Comment (robert-zaremba) [reply packages/algob/test/internal/core/runtime-environment.ts:318]: We should have a test to check if a default network is selected, when a user doesn't specify any network.

**Review by robert-zaremba — CHANGES_REQUESTED** [review]

File: packages/algob/test/internal/core/runtime-environment.ts (hunk 1 above)
Comment (amityadav0) [reply packages/algob/test/internal/core/runtime-environment.ts:318]: we cannot test this here, as the code where it happens is in `gatherArguments`
https://github.com/scale-it/algorand-builder/blob/4aa277d437bb70aa947d81c43cac0610b7e0fda2/packages/algob/src/internal/cli/cli.ts#L80

i am adding the test for the same in `arguments-parser.ts` test

Comment (amityadav0) [reply packages/algob/test/internal/core/runtime-environment.ts:318]: this code firsts load default 
https://github.com/scale-it/algorand-builder/blob/4aa277d437bb70aa947d81c43cac0610b7e0fda2/packages/algob/src/internal/core/params/builder-params.ts#L4

and then change Arguments as per cli input from user

**Review by robert-zaremba — APPROVED** [review]

**Comment by robert-zaremba** [comment]
Thanks for adding a new test. :+1:

**Pull request merged**